scripts/xml_to_csv/xml_to_csv.py

```diff
@@ -0,0 +1,40 @@
+# Written By: Jodi Gunawan
+# Date: Nov 15th, 2018
+# Description:  This script creates csv file from xml input file under the file train_labels
+
+import os
+import glob
+import pandas as pd
+import xml.etree.ElementTree as ET
+
+
+def xml_to_csv(path):
+    xml_list = []
+    for xml_file in glob.glob(path + '/*.xml'):
+        tree = ET.parse(xml_file)
+        root = tree.getroot()
+        for member in root.findall('object'):
+            value = (root.find('filename').text,
+                     int(root.find('size')[0].text),
+                     int(root.find('size')[1].text),
+                     member[0].text,
+                     int(member[4][0].text),
+                     int(member[4][1].text),
+                     int(member[4][2].text),
+                     int(member[4][3].text)
+                     )
+            xml_list.append(value)
+    column_name = ['filename', 'width', 'height', 'class', 'xmin', 'ymin', 'xmax', 'ymax']
+    xml_df = pd.DataFrame(xml_list, columns=column_name)
+    return xml_df
+
+
+def main():
+    for directory in ['train_labels']:
```

Comment:
Could we add an argument parser that allowed us to call this script for train or test images? That would allow us to look in either the train_images or test_images depending on if we wanted to have some test images in the future, thanks!

Reply:
https://docs.python.org/2/howto/argparse.html